tutorials/klusterai-api/rag.ipynb

```diff
@@ -0,0 +1,846 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "566e7e41",
+   "metadata": {},
+   "source": [
+    "# Retrieval-augmented generation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8566fe2a",
+   "metadata": {},
+   "source": [
+    "There are use cases when models struggle to complete tasks due to lack of specialized knowledge. This could lead to issues like hallucinations where models generate incorrect or fabricated information.\n",
+    "To help mitigate this, **Retrieval-Augmented Generation (RAG)** allows the model to pull real-time data from specified sources.\n",
+    "\n",
+    "This tutorial demonstrates how to use the <a href=\"https://docs.kluster.ai/api-reference/reference/#/http/api-endpoints/embeddings/v1-embeddings-post\" target=\"_blank\">embeddings endpoint</a> in a RAG pipeline with PDF document support using <a href=\"https://docs.llamaindex.ai/en/stable/module_guides/models/embeddings/\" target=\"_blank\">LlamaIndex</a> to handle the data and retrieval. \n",
+    "\n",
+    "\n",
+    "**Models:**\n",
```

Comment:
What about these models? Are these the models being used in the guide? Are they supported models? A little bit more context here would be helpful